DigitalLearningSolutions.Web.Tests/Controllers/Login/LoginControllerTests.cs

```diff
@@ -56,20 +59,20 @@ public void Index_should_render_basic_form()
         public void Index_should_redirect_if_user_is_authenticated()
         {
             // Given
-            var controllerWithAuthenticatedUser = new LoginController(loginService, userService, sessionService)
+            var controllerWithAuthenticatedUser = new LoginController(loginService, userService, sessionService, logger)
                 .WithDefaultContext()
                 .WithMockUser(true);
 
             // When
-            var result = controllerWithAuthenticatedUser.Index();
+            var result = controllerWithAuthenticatedUser.Index("");
```

Comment:
Is this required?

Reply:
It complains when I don't do it, and a simple null is ambiguous between the GET and POST methods. I suppose `string.Empty` would look a little neater though.

Reply:
You should be able to do this in the actual controller params, which will give the returnUrl a default value if not provided - (string? returnUrl = null)

Then we should be able to do Index() in these tests. :)